packages/central-server/src/tests/apiV2/submitMeditrakSurveyResponse.test.js

```diff
@@ -92,15 +83,6 @@ describe('POST /surveyResponse', async () => {
 
   before(async () => {
     defaultTimezone = (await models.database.getTimezone()).TimeZone;
-    sinon.stub(S3Client.prototype, 'uploadImage').callsFake(data => {
-      const id = generateId();
-      mockS3Bucket[id] = data;
-      return id;
-    });
-  });
```

Comment:
Ah yeah it's odd, seems like it should work. I was thinking of leaving these tests all commented out since we're doing a lot of manual testing around this, but yeah we should probably fix it instead. I'll have a look today.

Reply:
I remember now, sinon doesn't do mocking of module imports:
> Sinon is a stubbing library, not a module interception library
https://sinonjs.org/how-to/stub-dependency/

I had a look at the suggested addon packages that enable this, but I didn't want to spend any time on it since we should just move central-server to jest.

I also had a go at converting it all to jest, it's doable, especially with some code generator/migration tools, but it's still a big task and it went too far out of scope.

Happy for us to leave this for later?